selfdrive/ui/qt/offroad/wifiManager.cc

```diff
@@ -392,6 +403,27 @@ void WifiManager::disconnect() {
   }
 }
 
+QString WifiManager::ssid_from_path(const QDBusObjectPath &path) {
```

Comment:
will make other functions that call list_connections use this function in another PR